src/pages/public/HelperRegister.tsx

```diff
@@ -0,0 +1,358 @@
+              </div>
+              <p className="inline-block font-medium">{text}</p>
+            </li>
+          ))}
+        </ul>
+      </>
+    )
+  },
+
+  {
+    stepIndex: 1,
+    Content: ({ setFieldValue, values }: PassedFormProps) => (
+      <>
+        {[
+          ["account.firstName", "Vorname"],
+          ["account.lastName", "Nachname"],
+          ["account.phone", "Telefonnummer"],
+          ["account.email", "E-Mail"],
+          ["account.password", "Passwort"],
+          ["account.password_confirmation", "Passwort wiederholen"]
+        ].map(([key, label]) => (
+          <TextField
+            key={key}
+            label={label}
+            className="mb-4 text-brand"
+            fullWidth
+            onChange={(e) => setFieldValue(key, e.currentTarget.value)}
+          />
+        ))}
+        <div className="mb-2">
+          <FormTitle as="h2" className="mb-2">
+            Du bist
+          </FormTitle>
+          <div>
+            {[
+              ["Schüler*in", "Schüler*in"],
+              ["Student*in", "Student*in"],
+              ["Arbeitslos", "Arbeitslos"],
+              ["Beschäftigt", "Beschäftigt"]
+            ].map(([label, value], i) => {
+              const checked = value === values.employmentStatus;
+
+              return (
+                <Radio
+                  key={i}
+                  checked={checked}
+                  onClick={() => setFieldValue("employmentStatus", value)}
+                  block
+                >
+                  {label}
+                </Radio>
+              );
+            })}
+          </div>
+        </div>
+      </>
+    )
+  },
+  {
+    stepIndex: 2,
+    Content: ({ setFieldValue, values }: PassedFormProps) => (
+      <>
+        <div className="mb-2">
+          <FormTitle as="h2" className="mb-2">
+            Du suchst etwas für
+          </FormTitle>
+          <div>
+            <Checkbox
+              className="mb-2"
+              checked={values.fullTime === true}
+              onChange={() => setFieldValue("fullTime", true)}
+              block
+            >
+              Vollzeit
+            </Checkbox>
+            <Checkbox
+              checked={values.partTime === true}
+              onChange={() => setFieldValue("partTime", false)}
+              block
+            >
+              Teilzeit
+            </Checkbox>
+          </div>
+        </div>
+        <div className="mb-2">
+          <FormTitle as="h2" className="mb-2">
+            Für die Anreise bist du
+          </FormTitle>
+          <div>
+            <Radio
+              className="mb-2"
+              checked={values.pickupRequired === false}
+              onClick={() => setFieldValue("pickupRequired", false)}
+              block
+            >
+              Eigenständig
+            </Radio>
+            <Radio
+              checked={values.pickupRequired === true}
+              onClick={() => setFieldValue("pickupRequired", true)}
```

Comment:
Selber Fehler wie oben.
```suggestion
              onChange={() => setFieldValue("pickupRequired", true)}
```